.evergreen/config.yml

```diff
@@ -2341,12 +2368,30 @@ buildvariants:
     platform:
       # OSes that support versions of MongoDB>=2.6 with SSL.
       - awslinux
+    auth-ssl: "*"
+  display_name: "${platform} ${auth-ssl}"
+  tasks: &aws-server-versions
+    - ".5.0"
+    - ".4.4"
+    - ".4.2"
+    - ".4.0"
+    - ".3.6"
+    - ".3.4"
+    - ".3.2"
+    - ".3.0"
+    - ".2.6"
+
+- matrix_name: "tests-all-rhel70"
+  matrix_spec:
+    platform:
+      # OSes that support versions of MongoDB>=2.6 with SSL.
       - rhel70
     auth-ssl: "*"
   display_name: "${platform} ${auth-ssl}"
-  tasks: &all-server-versions
+  tasks: &rhel-server-versions
```

Comment:
To avoid adding extra unneeded testing can this just be 6.0, rapid, latest? Then we'd keep testing 2.6-5.0 on awslinx and move 6.0+ to rhel 7.

Reply:
Done